### DISCUSSION TITLE
Does forcing filetree naming scheme to be title case make sense?

### DISCUSSION THREAD

**Discussion by AliciaBytes** [oleeskild/obsidian-digital-garden#297] [General]
I wanted to start a discussion about this because it's not an issue itself, but rather a very opinionated default choice that might be worth discussing.

So currently all the folder names in the file tree are always getting converted to title case. My question is if this makes sense to be forced or if we should just keep the naming the way the user named them.

The main issue for me is that I have multiple folder names that are acronyms (e.g. TTRPG, HTTP, SQL, etc.) and title case looks weird for them. Ttrpg, Http, Sql look weird when you're used to the capitalized acronyms.

**Comment by oleeskild** [comment]
I very much agree with you. The user should be able to decide the casing for themselves. This was added in as part of a large PR, where I simply hadn't caught that this behaviour was added as well. The latest release of the template has removed this, so that your original casing is always used. 😄
